examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
OK, I got a TRIUX .fif which should be nicer as an example. Should it go under `misc` in mne-testing-data? Also it's about 500 megs, so should I truncate to couple of channels?